pkg/proxy/healthcheck/proxier_health.go

```diff
@@ -80,19 +85,19 @@ func newProxierHealthServer(listener listener, httpServerFactory httpServerFacto
 }
 
 // Updated updates the lastUpdated timestamp.
-func (hs *ProxierHealthServer) Updated() {
+func (hs proxierHealthServer) Updated() {
```

Comment:
hs is a *copy* of the receiver, this is effectively doing

x := 1
y := x
y++

// what is the value of x?

Reply:
but you want to leave the `*` on all references to `proxierHealthServer`